### ISSUE TITLE
about audio

### ISSUE THREAD

**Issue by Northwemko** [FroscoInnovativeTechnologies/VitaDockLite#1]
i check file content ,so i'm curious. how is the audio output?bluetooth?

**Comment by NekoMimiOfficial** [comment]
Unfortunately due to not having a plugin to send the vita's audio via USB I have no way of implementing this
You can export the audio via Bluetooth or a cable and unfortunately I don't have a way to make use of these as my laptop doesn't have Bluetooth support or a sound card capable of accepting an audio device as source

If anyone would like to help me out on this id be very happy
For now most people perhaps just hook up their vitas to a speaker anyways but if I can find someone to aid me in implementing a native audio support then please do (if any would like to contact me my discord is NekoMimi#7225)

**Comment by NekoMimiOfficial** [comment]
-

**Comment by Northwemko** [comment]
> Unfortunately due to not having a plugin to send the vita's audio via USB I have no way of implementing this You can export the audio via Bluetooth or a cable and unfortunately I don't have a way to make use of these as my laptop doesn't have Bluetooth support or a sound card capable of accepting an audio device as source
> 
> If anyone would like to help me out on this id be very happy For now most people perhaps just hook up their vitas to a speaker anyways but if I can find someone to aid me in implementing a native audio support then please do (if any would like to contact me my discord is NekoMimi#7225)

i can help u,bluetooth output audio only 2 requirements can work,do u have discord?this is more convenient for communication

**Comment by NekoMimiOfficial** [comment]
Yes I already sent my discord name and tag in the previous comment (NekoMimi#7225)

**Comment by NekoMimiOfficial** [comment]
Please note ,a regular Bluetooth card isn't gonna work , we need a Bluetooth receiver (capable of acting as a BT speaker)
